library/alloc/src/sync/arc.rs

```diff
@@ -239,7 +231,7 @@ impl<T: ?Sized + Unsize<U>, U: ?Sized> CoerceUnsized<Arc<U>> for Arc<T> {}
 impl<T: ?Sized + Unsize<U>, U: ?Sized> DispatchFromDyn<Arc<U>> for Arc<T> {}
 
 impl<T: ?Sized> Arc<T> {
-    fn from_inner(ptr: NonNull<ArcInner<T>>) -> Self {
+    pub(crate) fn from_inner(ptr: NonNull<ArcInner<T>>) -> Self {
```

Comment:
Should these be `pub(super)`?

Reply:
possibly! -- I wasn't sure whether that would be possible since the sibling module wants to get access, not the parent; but I'll try it -- never knew!

Reply:
Ah yes, `pub(super)` will do that for you 🙂 Anything that’s visible to a module is always visible to its children too, so siblings can peek at `pub(super)` items.